src/app/pages/BridgeDepositPage/components/ReturnToPortfolio/index.tsx

```diff
@@ -18,8 +17,9 @@ import {
   getWalletName,
   getWalletImage,
 } from 'app/components/UserAssets/TxDialog/WalletLogo';
+import { addNetworkByChainId } from '../../../../../utils/metamaskHelpers';
 
-const addNetworkCallback = isMainnet ? addRskMainnet : addRskTestnet;
+const chainId = isMainnet ? ChainId.RSK_MAINNET : ChainId.BSC_TESTNET;
```

Comment:
Shouldn't you have here `ChainId.RSK_TESTNET` instead of `ChainId.BSC_TESTNET`?

Reply:
Ohh good catch